/**:
  ros__parameters:
    stddev_thresholds:
      # Threshold value for the range in which GNSS position standard deviation is most reliable
      gnss_stddev_reliable_max: 0.1      #meters

      # Threshold value at which GNSS position standard deviation is not considered reliable
      gnss_stddev_unreliable_min: 0.25   #meters

      # Threshold value for yaw standard deviation
      yaw_stddev_deg_threshold: 0.3      #degrees

    validation:
      # Acceptable delay duration for GNSS pose data [sec]
      gnss_pose_timeout_sec: 1.0  #seconds

    debug:
      enable_debug_topics: true
